lms/djangoapps/teams/csv.py

```diff
@@ -2,6 +2,15 @@
 CSV processing and generation utilities for Teams LMS app.
 """
 
+import csv
+from django.contrib.auth.models import User
+from student.models import CourseEnrollment
+from xmodule.modulestore.django import modulestore
+
+from lms.djangoapps.teams.models import CourseTeam, CourseTeamMembership
+from .errors import AlreadyOnTeamInCourse, NotEnrolledInCourseForTeam
+from .utils import emit_team_event
+
 
 def load_team_membership_csv(course, response):
```

Comment:
This can be deleted if we're not using it.

Reply:
Which one are you specifically referring to? NotEnrolled is gone from the file?

Reply:
the `load_team_membership_csv()` function - that was just a stand-in before, and we're not using it now.

Reply:
I think this is for CSV export - Jansen's story. Let me know if it should still be removed.

Reply:
(I'm using this)